extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -125,6 +125,10 @@ class AmpSocialShare extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    if (!this.shareEndpoint_) {
+      return Promise.resolve();
```

Comment:
No user error/warning?

Reply:
Added a comment to this effect -- the endpoint throws an error if it gets far enough to check it in `buildCallback`. This case handles when `buildCallback` returns early after the component hides itself, in which case the internal variable is `null` for not getting to be resolved. A warning is thrown in one of the hiding cases, and I don't think we need to add an error where this case didn't have one before.